src/routes/(settings)/settings/billing/success/credits/+page.svelte

```
@@ -0,0 +1,106 @@
+<script lang="ts">
+  import { Button } from "$lib/components/ui/button";
+  import { CheckCircle, Zap, ArrowRight, Coins } from "@lucide/svelte";
```

ESLint

[error] 'Zap' is defined but never used.

(@typescript-eslint/no-unused-vars)